src/i18n/locale/en.json

```diff
@@ -269,10 +269,10 @@
       },
       "sort": {
         "label": {
-          "alphabetically": "Alphabetically",
+          "alphabetically": "A-Z",
```

Comment:
why

Reply:
Simpler and more immediate. Alphabetical is a concept that relates to a direction of the alphabet, which is also a concept that must be known beforehand.